settings/src/components/screen-navigation.js

```diff
@@ -32,7 +32,8 @@ const ScreenNavigation = ( { screen, children } ) => (
 				{ screen
 					.replace( '-', ' ' )
 					.replace( 'totp', 'Two-Factor Authentication' )
-					.replace( 'webauthn', 'Two-Factor Security Key' ) }
+					.replace( 'webauthn', 'Two-Factor Security Key' )
```

Comment:
Is 'Two-Factor Security Key' the usual term? Should it mention Hardware or Physical Key in the name?

Reply:
This is the existing text, and I agree a little funky. The text is the heading of the sub-screens, ie. click into the security keys section and that's this code.